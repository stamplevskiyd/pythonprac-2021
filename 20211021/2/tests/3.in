:is_small_factorial x x in (1, 2, 6, 20, 120, 720, 5040)
:output x print("__".join(list(x)))
is_small_factorial 120
is_small_factorial 5
output ("some text").upper()
:x t print(['#' * t[0] for i in range(t[1])])
x (5,4)
quit quit

