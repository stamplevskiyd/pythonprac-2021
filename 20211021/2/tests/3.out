2.0
3
4.0
5
6.0
"2:quit7"
